packages/opentelemetry-plugin-http/src/http.ts

```diff
@@ -314,7 +316,10 @@ export class HttpPlugin extends BasePlugin<Http> {
           span.end();
           return returned;
         };
-        return original.apply(this, [event, ...args]);
+
+        return plugin._safeExecute(span, () =>
+          original.apply(this, [event, ...args])
```

Comment:
shouldn't we return what `original.apply(this, [event, ...args])` given us here ?

Reply:
it will if no exception : https://github.com/open-telemetry/opentelemetry-js/blob/1327ff8878a7a729efe03fa31e84888804bf9a6a/packages/opentelemetry-plugin-http/src/http.ts#L401